src/main/java/com/exasol/adapter/dynamodb/mapping/AbstractColumnMapping.java

```diff
@@ -20,4 +20,21 @@ protected AbstractColumnMapping(final String exasolColumnName) {
     public final String getExasolColumnName() {
         return this.exasolColumnName;
     }
+
+    @Override
+    public boolean equals(final Object other) {
+        if (this == other) {
+            return true;
+        }
```

Comment:
It is usually good practice to check the isInstanceOf using the interface the class implements.